src/parser.pegjs

```diff
@@ -46,16 +46,42 @@
 // ---- Syntactic Grammar -----
 
 Grammar
-  = __ topLevelInitializer:(@TopLevelInitializer __)? initializer:(@Initializer __)? rules:(@Rule __)+ {
+  = __ imports:(@Import __)* topLevelInitializer:(@TopLevelInitializer __)? initializer:(@Initializer __)? rules:(@Rule __)+ {
```

Comment:
Imports, top-level initializer and initializer have to be in strict order, and it might be not that obvious, what it is. Can we just consider all of the things mentioned in this rule as statements, and parse them in no particular order?

Reply:
I think, it is possible, although I would still not use such flexibility and would not recommend that anyone do this. Current order reflects order in which statements will be executed:
- importing grammars (in specified order)
- running global initializer
- running per-parse initializer

Reply:
I assume the imported grammars effectively go at the *end* of the AST?  You usually don't want them to be the default start rule, and if you do, you write a pass-through rule as your first rule.

Reply:
Current PR defines only syntax, not semantics. You free to choose what to do with that AST. Currently AST only contains information, that it referenced to some external grammar. The pass that will deal with that information is to be written.

One of possible implementation of imports (which, for example, I've made for myself [almost ten years ago](https://github.com/Mingun/pegjs/tree/include-pass)) is to include them into one big AST; of course it is better to logically include new rules at the end of AST.